cpp/src/structure/graph.cu

```diff
@@ -36,7 +36,7 @@ void degree_from_offsets(vertex_t number_of_vertices,
 }
 
 template <typename vertex_t, typename edge_t>
-void degree_from_vertex_ids(const cugraph::experimental::Comm &comm,
+void degree_from_vertex_ids(const raft::handle_t *handle,
```

Comment:
I believe it is passed by reference in cuML. 
We may want to do that for consistency or does it have to be a pointer?

Reply:
updated